### PR TITLE
625 Add monai bundle example

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
This PR is related to #625 , a `hello-monai-bundle` example is added, and the readme file of `hello-monai` is updated.
These two examples are all tested with the latest nvflare 2.1.2

### CODE REVIEW

**Comment by yiheng-wang-nv** [comment]
Hi @YuanTingHsieh , could you please help to check this PR, thanks!
cc @Nic-Ma

File: examples/hello-monai-bundle/custom/monai_trainer.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
```
Comment (YuanTingHsieh) [inline examples/hello-monai-bundle/custom/monai_trainer.py:1]: not sure if this matters, 
can you update to 
```# Copyright (c) 2021-2022, NVIDIA CORPORATION.```

File: examples/hello-monai-bundle/config/config_fed_server.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      "id": "aggregator",
+      "name": "AccumulateWeightedAggregator",
+      "args": {
+        "aggregation_weights": {
+          "org1-a": 1.0,
+          "org1-b": 0.5
+        },
+        "expected_data_kind": "WEIGHTS"
+      }
+    }
```
Comment (YuanTingHsieh) [inline examples/hello-monai-bundle/config/config_fed_server.json:43]: can you update this part to

might also want to update the hello-monai example

```

    {
      "id": "aggregator",
      "path": "nvflare.app_common.aggregators.intime_accumulate_model_aggregator.InTimeAccumulateWeightedAggregator",
      "args": {
        "expected_data_kind": "WEIGHTS"
      }
    }

```

**Comment by YuanTingHsieh** [comment]
@yiheng-wang-nv can you also update both hello-monai examples from "App" structure to "Job" structure?

Basically just adding "meta.json" and introduce a new directory call "app".

You can check similar changes in: https://github.com/NVIDIA/NVFlare/pull/724

**Comment by yiheng-wang-nv** [comment]
> @yiheng-wang-nv can you also update both hello-monai examples from "App" structure to "Job" structure?
> 
> Basically just adding "meta.json" and introduce a new directory call "app".
> 
> You can check similar changes in: #724

Hi @YuanTingHsieh , I changed but met the following error when type in: `submit_job hello-monai-bundle`
```
Error: No other site can be specified if @ALL is used for job hello-monai-bundle
```
Does the command need to be changed?

**Comment by YuanTingHsieh** [comment]
> > @yiheng-wang-nv can you also update both hello-monai examples from "App" structure to "Job" structure?
> > Basically just adding "meta.json" and introduce a new directory call "app".
> > You can check similar changes in: #724
> 
> Hi @YuanTingHsieh , I changed but met the following error when type in: `submit_job hello-monai-bundle`
> 
> ```
> Error: No other site can be specified if @ALL is used for job hello-monai-bundle
> ```
> 
> Does the command need to be changed?

Thanks @yiheng-wang-nv for checking.
I made a mistake there.
I have fixed it in that PR.
Can you please check the changes in the PR again and apply it on your PR here?
Thanks

**Comment by YuanTingHsieh** [comment]
basically the meta.json would be something like:


```
{
  "name": "hello-monai",
  "resource_spec": {},
  "min_clients" : 2,
  "deploy_map": {
    "app": [
      "@ALL"
    ]
  }
}
```

**Review by YuanTingHsieh — CHANGES_REQUESTED** [review]
code LGTM, can you change the structure to "JOB" like in PR https://github.com/NVIDIA/NVFlare/pull/724

Thanks

**Comment by yiheng-wang-nv** [comment]
Hi @YuanTingHsieh , thanks! I updated these two examples and tested locally.

**Review by YuanTingHsieh — APPROVED** [review]
Thanks for this PR. LGTM

**Pull request merged**